src/vault/TimeRangeAuthorizer.sol

```diff
@@ -5,33 +5,54 @@ import "../interfaces/IAuthorize.sol";
 
 /**
  * @title TimeRangeAuthorizer
- * @dev Authorizer contract that allows actions only within a specified time range.
+ * @dev Authorizer contract that allows actions only outside a specified time range.
  */
 contract TimeRangeAuthorizer is IAuthorize {
-    address public owner;
```

Comment:
Maybe owner just to change the time was unecessary
since we could have the owner on the vault which can change authorizer ?